doc/tutorials/dicom_json.rst

```diff
@@ -0,0 +1,97 @@
+============================
+Introduction to JSON support
+============================
+
+.. versionadded:: 1.3
+
+Starting in *pydicom* version 1.3, some support for converting DICOM data to
+and from JSON format has been added. This support is considered to be in
+beta state, and the API is still subject to change.
```

Comment:
Not sure if beta state is ok, or even pre-release...

Reply:
No one has complained so far ... I think it should stay reasonably stable so beta sounds good to me.

Reply:
And with this documentation, maybe some more people will make use of it and there will be more feedback.